### ISSUE TITLE
Windows init does not work

### ISSUE THREAD

**Issue by dustinsoftware** [kentcdodds/react-fundamentals#7]
![image](https://user-images.githubusercontent.com/942358/75829412-1b165e00-5d7c-11ea-95c8-b5053df7f7b5.png)

**Comment by kentcdodds** [comment]
Thanks @dustinsoftware! Could you copy what's in that .log file shown at the bottom of the screenshot?

**Comment by dustinsoftware** [comment]
<details>
0 info it worked if it ends with ok
1 verbose cli [
1 verbose cli   'C:\\Users\\dustin.masters\\AppData\\Local\\Volta\\tools\\image\\node\\12.14.1\\6.13.4\\node.exe',
1 verbose cli   'C:\\Users\\dustin.masters\\AppData\\Local\\Volta\\tools\\image\\node\\12.14.1\\6.13.4\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'install',
1 verbose cli   'git+https://gist.github.com/bb452ffe53a5caa3600197e1d8005733.git',
1 verbose cli   '--global',
1 verbose cli   '--prefix',
1 verbose cli   'C:\\Users\\dustin.masters\\AppData\\Roaming\\npm-cache\\_npx\\7668',
1 verbose cli   '--loglevel',
1 verbose cli   'error',
1 verbose cli   '--json'
1 verbose cli ]
2 info using npm@6.13.4
3 info using node@v12.14.1
4 verbose npm-session 3532f4b0fdb8c1be
5 silly install loadCurrentTree
6 silly install readGlobalPackageData
7 silly fetchPackageMetaData error for git+https://gist.github.com/bb452ffe53a5caa3600197e1d8005733.git Command failed: git submodule update -q --init --recursive
7 silly fetchPackageMetaData C:/Program Files/Git/mingw64/libexec/git-core\git-submodule: line 7: basename: command not found
7 silly fetchPackageMetaData C:/Program Files/Git/mingw64/libexec/git-core\git-submodule: line 7: sed: command not found
7 silly fetchPackageMetaData C:/Program Files/Git/mingw64/libexec/git-core\git-submodule: line 21: .: git-sh-setup: file not found
8 timing stage:rollbackFailedOptional Completed in 0ms
9 timing stage:runTopLevelLifecycles Completed in 1485ms
10 verbose stack Error: Command failed: git submodule update -q --init --recursive
10 verbose stack C:/Program Files/Git/mingw64/libexec/git-core\git-submodule: line 7: basename: command not found
10 verbose stack C:/Program Files/Git/mingw64/libexec/git-core\git-submodule: line 7: sed: command not found
10 verbose stack C:/Program Files/Git/mingw64/libexec/git-core\git-submodule: line 21: .: git-sh-setup: file not found
10 verbose stack
10 verbose stack     at ChildProcess.exithandler (child_process.js:295:12)
10 verbose stack     at ChildProcess.emit (events.js:223:5)
10 verbose stack     at maybeClose (internal/child_process.js:1021:16)
10 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)
11 verbose cwd C:\code\kcd\react-fundamentals
12 verbose Windows_NT 10.0.19041
13 verbose argv "C:\\Users\\dustin.masters\\AppData\\Local\\Volta\\tools\\image\\node\\12.14.1\\6.13.4\\node.exe" "C:\\Users\\dustin.masters\\AppData\\Local\\Volta\\tools\\image\\node\\12.14.1\\6.13.4\\node_modules\\npm\\bin\\npm-cli.js" "install" "git+https://gist.github.com/bb452ffe53a5caa3600197e1d8005733.git" "--global" "--prefix" "C:\\Users\\dustin.masters\\AppData\\Roaming\\npm-cache\\_npx\\7668" "--loglevel" "error" "--json"
14 verbose node v12.14.1
15 verbose npm  v6.13.4
16 error code 1
17 error Command failed: git submodule update -q --init --recursive
17 error C:/Program Files/Git/mingw64/libexec/git-core\git-submodule: line 7: basename: command not found
17 error C:/Program Files/Git/mingw64/libexec/git-core\git-submodule: line 7: sed: command not found
17 error C:/Program Files/Git/mingw64/libexec/git-core\git-submodule: line 21: .: git-sh-setup: file not found
18 verbose exit [ 1, true ]

</details>

**Comment by dustinsoftware** [comment]
Ok, I'm pretty sure this is an issue with `volta`, an NVM-like solution to automatically switch Node versions. That package does some fancy things to wrap the local paths.

I did not have this issue on a separate Windows workstation that has Node 12 installed without volta. I did however run into a _separate_ issue which I'll file in a moment. Closing this one since it goes away without Volta installed.
